src/channel.rs

```diff
@@ -295,6 +295,15 @@ impl ChannelBuilder {
         );
         self
     }
+    
+    /// If set to zero, disables use of http proxies.
+    pub fn enable_http_proxy(mut self, num: i32) -> ChannelBuilder {
```

Comment:
How about using `bool`?

Reply:
The Options enum only define Integer, String and Pointeur

Reply:
modification done